lib/modules/mongo/__tests__/index.js

```diff
@@ -12,18 +18,68 @@ describe('module - mongo', function () {
   });
 
   describe('logs', function () {
-    it('TODO write tests');
+    it('should pull logs from "meteor" vm', async done => {
+      sh.cd(path.resolve(__rootdir, 'tests/project-1'));
+
+      sh.exec('mup mongo setup && mup mongo start', {silent: true});
+      const out = sh.exec('mup mongo logs', {silent: true});
```

Comment:
Can be check for some log output. May be check some words which is common to MongoDB.